cdap-standalone/bin/cdap.bat

```diff
@@ -219,7 +221,7 @@ set class=co.cask.cdap.data2.transaction.TransactionManagerDebuggerMain
 REM Skip first parameter
 for /f "usebackq tokens=1*" %%i in (`echo %*`) DO @ set params=%%j
 
-"%JAVACMD%" %DEFAULT_JVM_OPTS% %HADOOP_HOME_OPTS% %TOKEN_FILE_OPTS% -classpath "%CLASSPATH%" %class% %params%
+"%JAVACMD%" %DEFAULT_JVM_OPTS% %HADOOP_HOME_OPTS% %HIVE_SCRATCH_DIR_OPTS% %TOKEN_FILE_OPTS% -classpath "%CLASSPATH%" %class% %params%

```

Comment:
Adding `HIVE_SCRATCH_DIR_OPTS` here is not required as this line executes `TransactionManagerDebuggerMain`